.github/workflows/publish.yml

```diff
@@ -0,0 +1,41 @@
+name: docker package publish
+
+on:
+  push:
+    tags:
+      "*.*"
+
+env:
+  TARGET_PLATFORMS: linux/amd64,linux/arm64
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-publish-docker-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push docker image
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          tags: ghcr.io/Shoobx/shoobx.mocks3:${{ github.ref_name }}, ghcr.io/Shoobx/shoobx.mocks3:latest
+          platforms: ${{ env.TARGET_PLATFORMS }}
```

Comment:
We probably need to assign a concrete version to the image, not just `:latest`. Are we doing it somewhere else?

Reply:
Yes. On this line we build  with two tags:
<container_name>:<triggered_tag> and <container_name>:latest

Reply:
Ah, missed that! Got it.